packages/eos-components/src/styles.scss

```diff
@@ -103,74 +105,72 @@ $carousel-indicator-transition:      all .6s ease !default;
 $carousel-image-width:    50%;
 $carousel-image-lg-width: 400px;
 
-@import "../../template-ui/src/overrides/styles.scss";
+// Copy of the Bootstrap mixin. Allows to pass a border color different than the text color.
+@mixin eos-button-outline-variant($color, $border-color: $color, $color-hover: color-yiq($color), $active-background: $color, $active-border: $color) {
+  color: $color;
+  border-color: $border-color;
 
-@import 'bootstrap/scss/bootstrap';
-@import 'bootstrap-vue/src/index';
+  @include hover() {
+    color: $color-hover;
+    background-color: $active-background;
+    border-color: $active-border;
+  }
 
-@import 'mixins';
+  &:focus,
+  &.focus {
+    box-shadow: 0 0 0 $btn-focus-width rgba($active-border, .5);
+  }
 
-// Change button dark outline variant:
-.btn-outline-dark {
-  $value: theme-color("dark");
-  @include eos-button-outline-variant($value, $color-hover: $value, $border-color: $gray-500, $active-background: transparent, $active-border: $primary);
-}
-// Change buttons variant:
-@each $color, $value in $theme-colors {
-  .btn-#{$color}-primary {
-    $hover-value: scale-color($value, $lightness: $hover-lightness);
-    @include button-variant(
-      $background: $value,
-      $border: $value,
-      $hover-background: $hover-value,
-      $active-background: $hover-value,
-      $active-border: $hover-value);
+  &.disabled,
+  &:disabled {
+    color: $color;
+    background-color: transparent;
   }
-  .btn-#{$color}-light {
-    @include button-variant(
-      $background: $light,
-      $border: $light,
-      $hover-border: $value);
+
+  &:not(:disabled):not(.disabled):active,
+  &:not(:disabled):not(.disabled).active,
+  .show > &.dropdown-toggle {
+    color: $color;
+    background-color: $active-background;
+    border-color: $active-border;
+
+    &:focus {
+      @if $enable-shadows and $btn-active-box-shadow != none {
+        @include box-shadow($btn-active-box-shadow, 0 0 0 $btn-focus-width rgba($active-border, .5));
+      } @else {
+        // Avoid using mixin so we can pass custom focus shadow properly
+        box-shadow: 0 0 0 $btn-focus-width rgba($active-border, .5);
+      }
+    }
   }
 }
-.btn-light {
-  color: $dark;
-}
 
-.close.text-light {
-  text-shadow: none;
-}
+// Allows reusing the same style in the header and in the navbar:
+@mixin navbar-background($header-height) {
+  background-color: $header-color;
+  background-attachment: fixed;
+  background-position: top center;
+  background-repeat: no-repeat;
 
-// Change header color:
-$header-color: scale-color($secondary, $lightness: 50%);
+  @include media-breakpoint-down(xxl) {
+    background-size: auto $header-height;
+  }
```

Comment:
This rule was in the removed mixins.scss but there is no xxl breakpoint anymore. You probably removed the file and then rebased. https://github.com/endlessm/kolibri-explore-plugin/pull/173/files#diff-24cf3bf40e034d42265157f36698d67dea616cb72bd8aaf39433869c9c33b367L48-L50